openapi/components/schemas/QuoteChangeOrder.yaml

```
@@ -0,0 +1,486 @@
+type: object
+title: QuoteChangeOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+  - orderId
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for change order items.
+      On accept quote interim invoice is issued and linked to `invoiceId`.
+      After the invoice is paid changed items are applied to an order with `orderId`.
```

suggestion: I think we don't need to state 'order with `orderId`'
all orders have an orderId (not sure if I'm misunderstanding something here)
```suggestion
      After the invoice is paid, 
      changed items are applied to the related order.
```